### PR TITLE
fix(Target Devices): Dynamically add Capability Mappings

### PR DESCRIPTION
**Pull request by pastaq**
- Selecting different target devices will dynamically add capability maps, based on the target device, ensuring every feature works out of the box. These will only be applied if the source capability has not already been mapped in the device profile.

### CODE REVIEW

File: core/global/platform.gd [hunk 1]
```diff
@@ -178,7 +178,12 @@ func _read_dmi() -> PLATFORM:
 		return PLATFORM.AOKZOE_GEN2
 
 	# ASUS
-	elif product_name.contains("ROG Ally RC71L") and vendor_name == "ASUSTeK COMPUTER INC.":
+	elif product_name in [
```
Comment (pastaq) [inline core/global/platform.gd:181]: I was testing on the Ally X and needed it to work. I can move these to a separate PR if desired.

**Comment by philmmanjaro** [comment]
Tested Inputplumber 2f55df51 combined with a21df1a7. seems to work so far.

**Review by ShadowApex — APPROVED** [review]

File: core/global/launch_manager.gd [hunk 2]
```diff
@@ -297,18 +298,33 @@ func set_app_gamepad_profile(app: RunningApp) -> void:
 
 ## Sets the gamepad profile for the running app with the given profile
 func set_gamepad_profile(path: String, target_gamepad: String = "") -> void:
+	# Discover the currently set gamepad to properly add additional capabilities based on that gamepad
+	var profile_modifier = target_gamepad
```
Comment (ShadowApex) [inline core/global/launch_manager.gd:302]:
```suggestion
	var profile_modifier := target_gamepad
```

File: core/global/launch_manager.gd [hunk 3]
```diff
@@ -297,18 +298,33 @@ func set_app_gamepad_profile(app: RunningApp) -> void:
...
+	if target_gamepad.is_empty():
+		profile_modifier = settings_manager.get_value("input", "gamepad_profile_target", InputPlumber.DEFAULT_GLOBAL_PROFILE) as String
```
Comment (ShadowApex) [inline core/global/launch_manager.gd:304]:
```suggestion
		profile_modifier = settings_manager.get_value("input", "gamepad_profile_target", "gamepad") as String
```

File: core/systems/input/events/event.gd [hunk 4]
```diff
@@ -32,6 +33,8 @@ static func from_event(godot_event: InputEvent) -> InputPlumberEvent:
 		return null
 	elif godot_event is InputEventAction:
 		return null
+	elif godot_event is InputEventAction:
+		return null
```
Comment (ShadowApex) [inline core/systems/input/events/event.gd:37]:
```suggestion
```

**Review by ShadowApex — CHANGES_REQUESTED** [review]
Few changes required

**Review by ShadowApex — APPROVED** [review]

**Pull request merged**